<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlickrDialogBase</class>
 <widget class="QDialog" name="FlickrDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flickr</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Flickr API key</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="apiKey">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose Table</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Database</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Table</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dbFileName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dbFilePicker">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tableName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose Area</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Latitude</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>North</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>South</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="north">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="south">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Longitude</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>East</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>West</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="east">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="west">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>630</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QTextBrowser" name="logBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="endDate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose csv file</string>
   </property>
   <widget class="QLineEdit" name="csvFileName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="csvFilePicker">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CSV File</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="removeVectorLayer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Layer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeImages">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>700</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Images</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveLogCheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>save log?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
